jung-api/src/test/java/edu/uci/ics/jung/graph/CTreeEquivalenceTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2014 The Guava Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Is that the correct license and copyright if it's going into JUNG?

Reply:
nope :)

Reply:
Indeed it isn't, good catch! I'll address this later today.